src/methods.cpp

```diff
@@ -1468,7 +1473,8 @@ PyObject* meth_flash_devices(PyObject* self, PyObject* args)
     PyObject* obj = NULL;
     PyObject* callback = NULL;
     PyObject* dict;
-    if (!PyArg_ParseTuple(args, arg_parse("OO|O:", __FUNCTION__), &obj, &dict, &callback)) {
+    int network_id = -1;
+    if (!PyArg_ParseTuple(args, arg_parse("OO|Oi:", __FUNCTION__), &obj, &dict, &callback, &network_id)) {
```

Comment:
missing iOptions?

Reply:
adding the options will be there my next push